biocrnpyler/__init__.py

```diff
@@ -1,47 +1,46 @@
-# __init__.py - initialization of biocrnpyler toolbox
-# RMM, 11 Aug 2018
+# All core classes
+from .core.polymer import *
+from .core.species import *
+from .core.compartment import *
+from .core.reaction import *
+from .core.propensities import *
+from .core.parameter import *
+from .core.mixture import *
+from .core.chemical_reaction_network import *
+# The core component class
+from .core.component import *
+# Library of all components
+from .components.basic import *
+from .components.dna.assembly import *
+from .components.dna.construct import *
+from .components.dna.cds import *
+from .components.dna.misc import *
+from .components.dna.promoter import *
+from .components.dna.rbs import *
+from .components.dna.terminator import *
+from .components.construct_explorer import *
+from .components.integrase_enumerator import *
+from .components.combinatorial_complex import *
+from .components.combinatorial_conformation import *
+from .components.component_enumerator import *
+from .components.membrane import *
 
-from .chemical_reaction_network import *
-from .component import *
```

Comment:
Rather that importing from each subdirectory from here, would it be possible to use a more recursive import structure?  So in this file you do `from .components import *` and then in `components/init.py` you import from `basic`, etc?  That would allow things to be more modular, since developers would not have to make changes to the global package when they are added a new component, mechanism, etc.